api.yml

```diff
@@ -185,66 +83,166 @@ paths:
           description: not supported by this implementation!
 components:
   schemas:
-    IssueRequest:
+    ConstructCredentialRequest:
       type: object
+      required:
+        - issuer
+        - vcTemplateReference
+        - claims
+        - output
       properties:
         issuer:
           type: string
-          description: Optional identifier for the issuer of the credential. If this is missing, it may be pre-configured or implied by other values in the request.
-          example: did:example:76e12ec712ebc6f1c221ebfeb1f
-        subject:
+          description: A reference to the issuer's pre-configured profile.
```

Comment:
This presumes that there is pre-configuration involved... which means we have to define what is preconfigured. It would be good to avoid adding to the discussion at this point. It also introduces the term "profile", which others in the ecosystem may have different definitions for... the more I read this PR, the more I think that we need to start the process of actually having a spec for this HTTP API. the OpenAPI stuff is failing to provide enough background for the thought process behind this API.